src/components/calcite-date/calcite-date.scss

```diff
@@ -11,15 +11,15 @@
 }
 
 :host([scale="s"]) {
-  max-width: 246px;
+  max-width: 216px;
 }
 
 :host([scale="m"]) {
-  max-width: 288px;
+  max-width: 286px;
 }
 
 :host([scale="l"]) {
-  max-width: 400px;
+  max-width: 398px;
```

Comment:
🤷‍♂️

Reply:
pixel perfect, @macandcheese , pixel perfect

<img width="497" alt="Screen Shot 2020-04-29 at 10 02 59 AM" src="https://user-images.githubusercontent.com/1031758/80624602-c2241880-8a00-11ea-830a-a9c7a09d45d6.png">

Reply:
Ahh, 365 days in a year, 365 pixels tall. 
![tenor](https://user-images.githubusercontent.com/4733155/80624845-1c24de00-8a01-11ea-974e-9979bc0ad56e.gif)

Reply:
will need to change every leap year to 366px